### ISSUE TITLE
Error when building signed app

### ISSUE THREAD

**Issue by ariona** [thebylito/react-native-navigation-bar-color#5]
```
/Users/anonim/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1feb56eb5c57622bf14f0d3b41eac93f/res/values-v26/values-v26.xml:9:5-12:13: AAPT: error: resource android:attr/colorError not found.

/Users/anonim/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1feb56eb5c57622bf14f0d3b41eac93f/res/values-v26/values-v26.xml:13:5-16:13: AAPT: error: resource android:attr/colorError not found.

/Users/anonim/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1feb56eb5c57622bf14f0d3b41eac93f/res/values-v26/values-v26.xml:17:5-93: AAPT: error: style attribute 'android:attr/keyboardNavigationCluster' not found.

/Users/anonim/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1feb56eb5c57622bf14f0d3b41eac93f/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontStyle not found.

/Users/anonim/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1feb56eb5c57622bf14f0d3b41eac93f/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/font not found.

/Users/anonim/.gradle/caches/transforms-1/files-1.1/appcompat-v7-27.1.1.aar/1feb56eb5c57622bf14f0d3b41eac93f/res/values/values.xml:251:5-69: AAPT: error: resource android:attr/fontWeight not found.

error: failed linking references.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-navigation-bar-color:verifyReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt
```

**Comment by thebylito** [comment]
Solved in 0.0.7

**Comment by vikas-chhabra** [comment]
@thebylito  How you solved it?
